bot/cogs/deployment.py

```diff
@@ -62,17 +62,12 @@ def __init__(self, bot: AutoShardedBot):
         response = await self.bot.http_session.get(STATUS_URL)
         data = await response.json()
 
-        embed = Embed(
-            title="Service status",
-            color=Colour.blurple()
-        )
+        embed = Embed(title="Service status", color=Colour.blurple())
 
         for obj in data:
             key, value = list(obj.items())[0]
 
-            embed.add_field(
-                name=key, value=value, inline=True
-            )
+            embed.add_field(name=key, value=value, inline=True)
```

Comment:
this wasn't even multiline enough for my liking in the first place. I would prefer this on five lines.